### ISSUE TITLE
Invariant Violation: RelayQL: Unexpected invocation at runtime. Again!

### ISSUE THREAD

**Issue by rterysen-openroad** [facebook/relay#1202]
Hello, 
I have been trying to research this and the answers i am finding doesn't apply to what i am trying to accomplish. 

All the answers I am finding are requiring the schema to be within the same project as the GraphQL api. 

example: #865, #71

I want my front end relay/react client to  call on  GraphQL from a separate code source.  It is a separate project and does not have access to the Schema file. (unless i symlink it. that is hacky IMO). 

The Error i am getting is```
  ```
Invariant Violation: RelayQL: Unexpected invocation at runtime. Either the Babel transform was not set up, or it failed to identify this call site. Make sure it is being used verbatim as `Relay.QL`.
[1]     at invariant (/Users/rterysen/projects/React-Relay-Boilerplate/node_modules/fbjs/lib/invariant.js:38:15)
[1]     at Object.RelayQL [as QL] (/Users/rterysen/projects/React-Relay-Boilerplate/node_modules/react-relay/lib/RelayQL.js:40:52)
[1]     at Viewer (routes.js:11:23)
[1]     at wrappedQuery (/Users/rterysen/projects/React-Relay-Boilerplate/node_modules/isomorphic-relay-router/node_modules/react-router-relay/lib/QueryAggregator.js:140:22)
[1]     at Object.Query (/Users/rterysen/projects/React-Relay-Boilerplate/node_modules/react-relay/lib/buildRQL.js:95:16)
[1]     at Object.instrumentedCallback [as Query] (/Users/rterysen/projects/React-Relay-Boilerplate/node_modules/react-relay/lib/RelayProfiler.js:137:40)
[1]     at /Users/rterysen/projects/React-Relay-Boilerplate/node_modules/react-relay/lib/getRelayQueries.js:56:36
[1]     at Array.forEach (native)
[1]     at Object.getRelayQueries (/Users/rterysen/projects/React-Relay-Boilerplate/node_modules/react-relay/lib/getRelayQueries.js:49:31)
[1]     at Object.instrumentedCallback [as getQueries] (/Users/rterysen/projects/React-Relay-Boilerplate/node_modules/react-relay/lib/RelayProfiler.js:137:40)
```

I have read that i need babelRelayPlugin.js to fix this. That plugin rely's on a schema that I didn't want in this project.  Has anyone written a React/Relay Client that uses a detached GraphQL API? 

```

Can anyone give me some advice on this?

**Comment by rterysen-openroad** [comment]
Found  way of grabbing query here. 

https://github.com/facebook/relay/issues/206#issuecomment-138186103

**Comment by jinxac** [comment]
I was trying out something and I bumped into this.

I created first login of my `react-native` app using react-native and redux. 

So initially my `.babelrc` was

```
{
  "presets": [
    "react-native"
  ]
}
```
After this I installed my packages using `npm install`. 
Now my post login functionality is built up in graphql, so I thought lets bring in Relay, so I changed my .babelrc to following:-
```

{
  "passPerPreset": true,
  "presets": [
    {
      "plugins": [
        "./plugins/babelRelayPlugin"
      ]
    },
    "react-native"
  ]
}
```

Now, when i compiled my app again I was hitting the issue.

I deleted the `node_modules` folder and ran `npm install` again post which it works fine.
